lcls_tools/common/data_analysis/projection_fit/gaussian_model.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from scipy.stats import norm, gamma
+from scipy.ndimage import gaussian_filter
+from lcls_tools.common.data_analysis.projection_fit.method_base import MethodBase
+
+
+class GaussianModel(MethodBase):
+    """
+    GaussianModel Class that finds initial param values for gaussian distribution
+        and builds probability density functions for the likelyhood a param
+        to be that value based on those initial param values
+
+    - passing this class the variable profile_data automatically updates
+        the initial values and and probability density functions to match that data
+    """
+
+    param_names: list = ["amplitude", "mean", "sigma", "offset"]
+    param_bounds: np.ndarray = np.array(
+        [[0.01, 1.0], [0.01, 1.0], [0.01, 5.0], [0.01, 1.0]]
+    )
+
+    def __init__(self, profile_data: np.ndarray = None):
+        if profile_data is not None:
+            self.profile_data = profile_data
+            self.find_init_values(self.profile_data)
+            self.find_priors(self.profile_data)
+
+    def find_init_values(self, data: np.array) -> np.array:
+        offset = float(np.min(data))
+        amplitude = np.max(gaussian_filter(data, sigma=5)) - offset
+        mean = np.argmax(gaussian_filter(data, sigma=5)) / (len(data))
+        sigma = 0.1
+        self.init_values_list = np.array([amplitude, mean, sigma, offset])
+        self.init_values = {"amplitude":amplitude,"mean":mean,"sigma":sigma,"offset":offset}
```

Comment:
it was requested that users might like the option to grab init values and not have to have knowledge of what parameters belonged to the ordered np.array. So self.init_values was changed from a list to a dictionary. However, to run scipy.optimize.minimize an np.array is still needed so that is saved in a different variable.

Reply:
we shouldn't save it in a different variable, we should just do the mapping inside the scipy.optimize.minimize call, should be a one-liner

Reply:
I apologize, I did nor realize python dictionaries are ordered.

Reply:
while technically dicts are ordered we don't have to rely on that, we can use 
`np.array([self.init_values[name] for name in self.param_names])`